articles/aks/learn/quick-windows-container-deploy-powershell.md

```diff
@@ -126,6 +126,18 @@ creating a node pool to run Windows Server containers, the default value for `-V
 [restricted VM sizes][restricted-vm-sizes]. The minimum recommended size is **Standard_D2s_v3**. The
 previous command also uses the default subnet in the default vnet created when running `New-AzAksCluster`.
 
+## Add a Windows Server 2019 or Windows Server 2022 node pool
+
+AKS supports Windows Server 2019 and Windows Server 2022 node pools. For Kubernetes versions "1.25.0" and higher, Windows Server 2022 is the default operating system. For earlier versions, Windows Server 2019 will be the default operating system. To use the non-default OS, you must specify the `OsSKU` to `Windows2019` or `Windows2022` except specifying `OsType` to `Windows`.
```

Comment:
I think we should change "except specifying" to "and specify" because we want customers to specify both

Reply:
Suggest rewording to - 

AKS supports Windows Server 2019 and 2022 node pools. For Kubernetes version 1.25.0 and higher, Windows Server 2022 is the default operating system. For earlier Kubernetes versions, Windows Server 2019 is the default OS. To use Windows Server 2019, you need to specify the following parameters:

* **osSKU** set the value to `Windows2019`
* **osType** set the value to `Windows`

Reply:
Thank you. Updated.